sdk/openai/openai/README.md

```diff
@@ -17,6 +18,7 @@ Checkout the following examples:
 - [Multiple Completions](#generate-multiple-completions-with-subscription-key)
 - [Chatbot](#generate-chatbot-response)
 - [Summarize Text](#summarize-text-with-completion)
+- [Analyze Business Data](#analyze-business-data)
```

Comment:
should we stick with "your own data" here so it maps to the service concept?

Reply:
This is a list of common scenarios not features. Although, "bring your own data" is descriptive, I think the scenario is more about analyzing the data rather than just bringing them.